src/ProtobufProtocolSupport.chpl

```diff
@@ -485,6 +485,8 @@ module ProtobufProtocolSupport {
     }
 
     proc enumAppend(val: uint(64), fieldNumber: int, ch: writingChannel) throws {
+      if val == 0 then return;
```

Comment:
It should say in the PR message that it adds avoidance of writing default value enums

Reply:
Sure, I will do that.